Tiltfile

```diff
@@ -153,21 +153,6 @@ def capz():
         deps = ["api", "cloud", "config", "controllers", "exp", "feature", "pkg", "go.mod", "go.sum", "main.go"]
     )
 
-    k8s_resource('capz-controller-manager:deployment:capz-system', objects=[
```

Comment:
Because it should not be present. When i make a code change on a running cluster, it would delete and recreate the exiting objects amcp, amc, ammp etc. which would trigger a delete on the cluster and the crds are also regenerated.